src/app/sales/sale.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Cake } from "../cakes/cake.interface";
+
+export interface Sale {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.